api/v1beta1/flowcollector_webhook.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"fmt"
+	"github.com/netobserv/network-observability-operator/api/v1alpha1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+// ConvertTo converts this v1alpha1 FlowCollector to its v1alpha1 equivalent (the conversion Hub)
+// https://book.kubebuilder.io/multiversion-tutorial/conversion.html
+func (r *FlowCollector) ConvertTo(dstRaw conversion.Hub) error {
+	dst := dstRaw.(*v1alpha1.FlowCollector)
+
+	if err := Convert_v1beta1_FlowCollectorSpec_To_v1alpha1_FlowCollectorSpec(&r.Spec, &dst.Spec, nil); err != nil {
+		return fmt.Errorf("copying v1alpha1.FlowCollector.Spec into v1alpha1.FlowCollector.Spec: %w", err)
```

Comment:
typo, it should be "v1beta1" into "v1alpha1" if I believe the name of the generated function.
There's the same typo in the above comment L26: "converts this v1alpha1 FlowCollector to its v1alpha1 equivalent"

Reply:
yep sorry will fix both typos